deploy/50_cloud-ingress-operator.Deployment.yaml

```diff
@@ -36,7 +36,7 @@ spec:
           env:
             # "" so that the cache can read objects outside its namespace
             - name: WATCH_NAMESPACE
-              value: ""
+              value: "openshift-sre-sshd,openshift-cloud-ingress-operator,openshift-ingress,openshift-ingress-operator,openshift-kube-apiserver"
```

Comment:
Does this work?? That's awesome, and way better.

Reply:
yeah https://github.com/openshift/cloud-ingress-operator/pull/177/files#diff-435eecb1d2af40ee96747f88ab71eb2758da989c92f76dcb1f714fc1b300c633R104 made this available :)